docs/GCE.md

```diff
@@ -1,72 +1,115 @@
+```
+Note: we disable automated reboots for this demo to avoid interrupting the instructions.
 
+Then create the cluster with the following command ("calico-1 calico-2 calico-3" is the list of nodes to create):
 ```
-ip addr add 10.240.10.1 peer 10.240.0.1 dev ens4v1
+gcloud compute instances create \
+  calico-1 calico-2 calico-3 \
+  --image-project coreos-cloud \
+  --image coreos-alpha-709-0-0-v20150611 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config.yaml
 ```
 
-There's more on this situation here, in case you want to understand this further [http://marc.info/?l=bird-users&m=139809577125938&w=2](http://marc.info/?l=bird-users&m=139809577125938&w=2)
+## Installing calicoctl on each node
+On each node, run these commands to set up Calico:
+```
+# Download calicoctl and make it executable:
+wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.5/calicoctl
+chmod +x ./calicoctl
 
-## Starting calico and running containers
-Now, you can just follow the standard getting started instructions for downloading calico and creating workloads. See https://github.com/Metaswitch/calico-docker/blob/master/docs/GettingStarted.md#installing-calico for more details.
+# Grab our private IP from the metadata service:
+export metadata_url="http://metadata.google.internal/computeMetadata/v1/"
+export private_ip=$(curl "$metadata_url/instance/network-interfaces/0/ip" -H "Metadata-Flavor: Google")
 
-Note that etcd should already be running on the master, core1 and core2 nodes, with data stored on the master node, and core1 and core2 run etcd in proxy mode, so no clustering is required.  Check this by running ```etcdctl ls /``` on each node.  If it is not running, then restart it by running ```docker start etcd```.
+# Start the calico node service:
+sudo ./calicoctl node --ip=$private_ip
 
-## (Optional) Enabling traffic from containers to the internet
-The test endpoints will be unable to access the internet - that is because the internal range we are using is not routable. Hence to get external connectivity, SNAT is called for using the following `iptables` rule (on both hosts).
+# Work-around a [BIRD routing issue](http://marc.info/?l=bird-users&m=139809577125938&w=2)
+# This tells BIRD that it's directly connected to the upstream GCE router.
+sudo ip addr add $private_ip peer 10.240.0.1 dev ens4v1
+```
+Then, on any one of the hosts, run this command to create an IP pool with IP-in-IP and NAT enabled:
+```
+./calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
+```
+IP-in-IP alows Calico to route traffic between containers.  NAT allows the containers to make outgoing connections to the internet.
 
+## Create a couple of containers and check connectivity
+On one host, run:
+```
+export DOCKER_HOST=localhost:2377
+docker run -e CALICO_IP=192.168.1.1 --name container-1 -tid busybox
+./calicoctl profile add TEST
+./calicoctl profile TEST member add container-1
```

Comment:
On a 2nd read of this, I wonder if it would be punchier to use "-e CALICO_PROFILE=test" on each node.